python/tidy/servo_tidy/tidy.py

```diff
@@ -81,6 +81,24 @@
     " accessible to\n// web pages."
 ]
 
+WHITELISTED_DEPENDENCIES = {
```

Comment:
We should really be moving this to the config file, but that shouldn't be a concern for this PR. I'll take care of it in an easy issue :smile: